Description: gnutls support for LZO compression (and gnutls extra) was
 removed upstream, stop trying to use it.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/638736
Last-Update: 2014-08-09

--- a/wmbiff/gnutls-common.c
+++ b/wmbiff/gnutls-common.c
@@ -4,7 +4,6 @@
 #include <stdlib.h>
 #include <string.h>
 #include <gnutls/gnutls.h>
-#include <gnutls/extra.h>
 #include <gnutls/x509.h>
 #include <gnutls/openpgp.h>
 #include <time.h>
@@ -509,7 +508,6 @@ void print_list(void)
 
 	printf("Compression methods:");
 	printf(" ZLIB");
-	printf(", LZO");
 	printf(", NULL\n");
 }
 
@@ -637,8 +635,6 @@ void parse_comp(char **comp, int ncomp,
 				comp_priority[j++] = GNUTLS_COMP_NULL;
 			if (strncasecmp(comp[i], "ZLI", 3) == 0)
 				comp_priority[j++] = GNUTLS_COMP_ZLIB;
-			if (strncasecmp(comp[i], "LZO", 3) == 0)
-				comp_priority[j++] = GNUTLS_COMP_LZO;
 		}
 		comp_priority[j] = 0;
 	}
